03-bash/task1.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+PARAM_N=1
+FILE_PATH=~/bash/task1.out
+LOCALE=US
+
+function usage()
+{
+    echo
+    echo "USAGE: task1.sh [-h|--help] [-n num] [file]"
+    echo 
+    echo "-h|--help:  this page,"
+    echo "-n num:     output files to be left"
+    echo "file:       the output file location, default ~/bash/task1.out"
+    echo
+}
+
+function printLocalizedMessages()
+{
+    if [[ "$LOCALE" == UA ]]; then
+    case "$1" in
+        "MSG1") echo "-n параметр повинен доривнювати або бути бiльше за одиницю" >&2;;
```

Comment:
In my opinion it is better to use some text instead of MSG1 e.g.  MSG_PARAMETER_LESS_THAN_1